soccer/ui/qt/MainWindow.ui

```diff
@@ -2198,7 +2198,7 @@
           <attribute name="title">
            <string>Testing Tab</string>
           </attribute>
-          <layout class="QVBoxLayout" name="verticalLayout_12">
+          <layout class="QVBoxLayout" name="verticalLayout_14">
```

Comment:
why these name changes?

Reply:
Oops? The warnings were bugging me (they're duplicated) so I fixed them and committed...

Reply:
Maybe a good opportunity to give them slightly more descriptive names?